### PR TITLE
feat: BREAKING remove /connectors; send them as metadata

### LINKED ISSUE #321: Explicitly define "recommended connectors" in ledger config
Currently, the connectors endpoint returns all registered connectors on the ledger.

This is unsafe. Anyone could create a connector and make it return the cheapest quotes. It's useful to be able to get a "default" set of connectors from the ledger. However that list must be curated, so for now lets add a config variable which allows the ledger operator to specify which connectors should be included in the `/connectors` endpoint. (For backwards compatibility, it's fine if `/connectors` returns all connectors by default for now.)

Proposed config:

``` sh
LEDGER_RECOMMENDED_CONNECTORS=connie,alice
LEDGER_RECOMMENDED_CONNECTORS=* # default
```

### PR DESCRIPTION
**Pull request by sentientwaffle**
Fixes #321

### CODE REVIEW

File: test/configSpec.js [hunk 1]
```diff
@@ -189,4 +189,23 @@ describe('loadConfig', () => {
...
     })
   })
+
+  describe('config.recommendedConnectors', () => {
+    it('defaults to undefined', () => {
+      const _config = loadConfig()
+      expect(_config.recommendedConnectors).to.equal(undefined)
+    })
+
+    it('is undefined when recommendedConnectors is "*"', () => {
```
Comment (emschwartz) [inline test/configSpec.js]: Does `*` mean everyone is recommended?

Comment (sentientwaffle) [inline test/configSpec.js]: All connector accounts. Its the current (and default) behavior.

Comment (emschwartz) [inline test/configSpec.js]: Got it. What happens if you configure `RECOMMENDED_CONNECTORS` with an account that doesn't have the `connector` field set?

Comment (sentientwaffle) [inline test/configSpec.js]: Any accounts listed in `RECOMMENDED_CONNECTORS` that aren't actually connector's will be excluded from the returned list. If they are later changed to connectors, they will get added in automatically.

**Comment by emschwartz** [comment]
LGTM once tests pass

**Pull request merged**